video/out/vo_dmabuf_wayland.c

```diff
@@ -191,6 +215,15 @@ static void draw_frame(struct vo *vo, struct vo_frame *frame)
             p->want_reset = false;
     }
 
+    // 2. adjust viewport source if needed
+    struct mp_rect src;
+    struct mp_rect dst;
+    struct mp_osd_res osd;
+    mp_get_src_dst_rects(NULL, vo->opts, vo->driver->caps, vo->params,
```

Comment:
You should pass a log here.

Reply:
I intentionally pass null because I don't want logging for each frame drawn

Reply:
Oh that's a fair point. Just out of curiosity, is there a way to rework this check so it doesn't have to run on every frame? I guess it's not a big deal.

Reply:
as image can change dimensions at any point in container (for example with mpeg ts) I think it needs to run every frame.